public/index.html

```diff
@@ -3,17 +3,57 @@
 <head>
   <title>Word Scramble</title>
   <script src="bower_components/angular/angular.min.js"></script>
+  <link rel="stylesheet" href="appStyle.css">
 </head>
-<body>
-  <div class="main-container">
-    <h1>Word Scramble</h1>
-    <div class="app" ng-app="scrambleApp">
-    Hello
-      <div ng-controller="hihi">Yellow</div>
-    </div>
-
-    <script src="scripts/app.js"></script>
-    <script src="scripts/controllers/input.js"></script>
+<body ng-app="scrambleApp">
+  <div class="container" ng-app="scrambleApp">
+    <header class="header">
+      <h1 class="header-title">Word Scramble</h1>
+    </header>
+    <main class="game-container" ng-controller="mainController">
+
+      <div scramble-input class="start-screen text-screen" ng-if="state==='start'" ng-controller="startController">
+        <div class="info">Try to unscramble to letters provided.</div>
+        <div class="info">Press <strong>Enter</strong> to start the game!</div>
+      </div>
+
+      <div scramble-input class="play-screen" ng-if="state==='play'" ng-controller="playController">
+        <div class="play-letter-box">
+          <span class="user-letter" ng-repeat="userLetter in userInput">
+            {{userLetter.letter}}
+          </span><!--
+          --><span class="scramble-letter" ng-repeat="letter in letters">
+            {{letter.letter}}
+          </span>
+        </div>
+        <div class="play-stats">
+          <div class="play-time info">Time remaining: <strong>{{timer}}</strong></div>
+          <div class="play-score info">Current score: <strong>{{score}}</strong></div>
+        </div>
+      </div>
+
+      <div scramble-input class="score-screen text-screen" ng-if="state==='score'" ng-controller="scoreController">
```

Comment:
same as above, would recommend ng-if="isScore()" or something, not sure what "state==='score'" means... what is score?